src/components/ebay-icon/template.marko

```diff
@@ -4,6 +4,7 @@
 </if>
 <else-if(data.isInline)>
     <svg w-bind ${data.htmlAttributes} ${data.accessibilityAttributes} focusable="false" class=data.classes>
+        <defs w-id="defs" w-body if(data.renderDefs)/>
```

Comment:
Can the symbol be directly included here? why do we need a transformer?

Reply:
@aselvaraj the thing is we want to make it so that you only get whichever symbols you use in the javascript bundle.

The transform for this implementation of `ebay-icon` does this:

```marko
 <ebay-icon type="inline" name="close"/>

 <!-- Becomes -->

<ebay-icon type="inline" name="close"><include('$DIRNAME/symbols/close.marko')/></ebay-icon>
```

Which basically just makes it so that you automatically include only the specific symbol you are asking for.